css/pages/home.css

```diff
@@ -271,6 +270,99 @@ margin: 10px 0;
 /* Express Functionality end */
 
 /* Easily communicate start */
+.easily-background-color{
+  background-color: #F6F9FE;
+}
+
+.container > .row > .tblock {
+    font-size: 25px;
+    font-weight: normal;
+    font-family: Roboto, 'Open Sans', BlinkMacSystemFont, 'Segoe UI','Helvetica Neue',Arial,'Noto Sans',sans-serif,'Apple Color Emoji','Segoe UI Emoji','Segoe UI Symbol','Noto Color Emoji';
+    color: black;
+    line-height: 1.5;
+    
+}
+
+.tblock .description {
+    font-family: Roboto, 'Open Sans', BlinkMacSystemFont, 'Segoe UI','Helvetica Neue',Arial,'Noto Sans',sans-serif,'Apple Color Emoji','Segoe UI Emoji','Segoe UI Symbol','Noto Color Emoji';
+    line-height: 1.7;
+    font-weight: 450;
+    color: #6C6C6C;
+    font-size: 15px;
+    margin-top: 15px;
+}
+
+.test .fa {
+    color: #7C4FE0;
+    margin-right:5px ;
+    /* line-height:2.5; */
+    
+}
+.test{
+    font-family: Roboto, 'Open Sans', BlinkMacSystemFont, 'Segoe UI','Helvetica Neue',Arial,'Noto Sans',sans-serif,'Apple Color Emoji','Segoe UI Emoji','Segoe UI Symbol','Noto Color Emoji';;
+    display: flex;
+    align-items: center;
+}
+
+.fa-li, li{
+    vertical-align: baseline;
+}
+
+
+
+.row > .tblock > .fa-ul {
```

Comment:
čia pridėjau tėvines klases, nes font-awesome perrašinėjo margin-left  .fa-ul klasei
![easily comunicate ml](https://user-images.githubusercontent.com/106968822/180596932-74eb9ed3-2fb8-47d4-a8be-68d643632056.png)
: